zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -200,48 +200,47 @@ public void buildAllPackages(List<HeliumPackage> pkgs) throws IOException {
     buildAllPackages(pkgs, false);
   }
 
-  public File getHeliumPackageDirectory(String pkgName) {
+  private File getHeliumPackageDirectory(String pkgName) {
     return new File(heliumBundleDirectory, pkgName);
   }
 
-  public File getHeliumPackageSourceDirectory(String pkgName) {
+  private File getHeliumPackageSourceDirectory(String pkgName) {
     return new File(heliumBundleDirectory, pkgName + "/" + HELIUM_BUNDLES_SRC_DIR);
   }
 
-  public File getHeliumPackageBundleCache(String pkgName) {
+  private File getHeliumPackageBundleCache(String pkgName) {
     return new File(heliumBundleDirectory, pkgName + "/" + HELIUM_BUNDLE_CACHE);
   }
 
-  public static List<String> unTgz(File tarFile, File directory) throws IOException {
-    List<String> result = new ArrayList<String>();
-    InputStream is = new FileInputStream(tarFile);
-    GzipCompressorInputStream gcis = new GzipCompressorInputStream(is);
-    TarArchiveInputStream in = new TarArchiveInputStream(gcis);
-    TarArchiveEntry entry = in.getNextTarEntry();
-    while (entry != null) {
-      if (entry.isDirectory()) {
+  private static List<String> unTgz(File tarFile, File directory) throws IOException {
+    List<String> result = new ArrayList<>();
```

Comment:
It's the same :) I mean, List<String> result = new ArrayList<>() implies the ArrayList is of type String. No need to code it.